src/cli.zig

```diff
@@ -184,7 +185,7 @@ pub const Arguments = struct {
         clap.parseParam("-e, --external <STR>...           Exclude module from transpilation (can use * wildcards). ex: -e react") catch unreachable,
         clap.parseParam("-l, --loader <STR>...             Parse files with .ext:loader, e.g. --loader .js:jsx. Valid loaders: js, jsx, ts, tsx, json, toml, text, file, wasm, napi") catch unreachable,
         clap.parseParam("-u, --origin <STR>                Rewrite import URLs to start with --origin. Default: \"\"") catch unreachable,
-        clap.parseParam("-p, --port <STR>                  Port to serve bun's dev server on. Default: \"3000\"") catch unreachable,
```

Comment:
the help text is confusing since there is no dev server now

Reply:
I noticed that but we should merge this. I was just fixing capitalization.

I can do a more thorough clean up of `bun dev` stuff tomorrow